src/nrepl/project-types.ts

```diff
@@ -152,11 +152,13 @@ async function leinProfilesAndAlias(defproject: any, connectSequence: ReplConnec
 const NREPL_VERSION = "0.6.0",
     CIDER_NREPL_VERSION = "0.23.0",
     PIGGIEBACK_VERSION = "0.4.2",
-    FIGWHEEL_MAIN_VERSION = "0.2.3";
+    FIGWHEEL_MAIN_VERSION = "0.2.3",
+    CLJ_KONDO_VERSION = "2020.04.05";
 
 const cliDependencies = {
     "nrepl": NREPL_VERSION,
     "cider/cider-nrepl": CIDER_NREPL_VERSION,
+    "clj-kondo": CLJ_KONDO_VERSION
```

Comment:
This should be added to lein and shadow dependencies as well, right?

Also, I wonder what the user should do if the project already has clj-kondo dependencies. Maybe we need to add an opt-out of Calva injecting it?

Reply:
Oh yeah, I forgot to go back and do that. Good catch, lol. I don't think it's going to affect anything if we inject it and it's already there, though I'll do some testing like adding a different version to the project deps and see what happens.

Reply:
Added it to the lein dependencies. I'm still unsure that it needs to go into shadow dependencies since that should just be cljs deps right? Since the debugger is just for clojure I figured that was not relevant.

Reply:
Also fyi, I tested with lein jack-in and a diff version of clj-kondo. I'm unsure which version (the injected one or the project one) gets actually used though. It seems unlikely any issues will arise from this, so I'm not sure that an opt-out makes sense to add. But maybe we can just keep it in mind and add it later if needed? I could be missing something though.

Reply:
Also I think if they wanted to opt out they could start their own repl aside from jack-in? This may be the way to go at least for now.

Reply:
A middle ground would be to have the clj-kondo version as a setting.